sphinx_selective_exclude/eager_only.py

```diff
@@ -31,9 +32,14 @@ def run(self, *args):
         env = self.state.document.settings.env
         env.app.builder.tags.add('TRUE')
         #print(repr(self.arguments[0]))
-        if not env.app.builder.tags.eval_condition(self.arguments[0]):
-            return []
-
+        try:
+            if not env.app.builder.tags.eval_condition(self.arguments[0]):
+                return []
+        except Exception as err:
+            logger = logging.getLogger(__name__)
+            logger.warning(('exception while evaluating only directive expression: %s'), err,
```

Comment:
This should be `"exception while evaluating 'only' directive expression"`, otherwise it's confusing. Parens around message are superfluous.

Reply:
I've done this